source/docs/03-metadata.xml

```diff
@@ -1614,6 +1614,13 @@
                               <specDesc key="typeDesc"/>
                            </specList>
                         </p>
+                        <p>The dating of printed sources is a relevant aspect for questions of provenance and edition. In the absence of bibliographical information, e.g. on the edition or the year of origin, plate numbers are an essential aid to dating. Though the name might suggest otherwise, plate numbers can be described as designations assigned to a resource by a music publisher which need not necessarily consist (only) of numbers. When present, they are usually printed at the bottom of each page of a musical print and sometimes appear on the title page as well. In MEI any such plate numbers can be encoded within the <gi scheme="MEI">plateNum</gi> element as plain text, similar to:</p>
```

Comment:
```suggestion
                        <p>The dating of printed sources is a relevant aspect for questions of provenance and edition. In the absence of bibliographical information, e.g. on the edition or the year of origin, plate numbers are an essential aid to dating. Though the name might suggest otherwise, plate numbers are designations assigned to a resource by a music publisher and need not consist (only) of numbers. When present, they are usually printed at the bottom of each page of a musical print and sometimes appear on the title page as well. In MEI any such plate numbers can be encoded within the <gi scheme="MEI">plateNum</gi> element as plain text, similar to:</p>
```